tooling/ci/ddtrace-info.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -eu
+
+echo "Loading install script"
+curl -Lf -o /tmp/dd-library-php-setup.php \
+  https://raw.githubusercontent.com/DataDog/dd-appsec-php/installer/dd-library-php-setup.php
+
+cd /binaries
+
+INSTALLER_ARGS=(--tracer-file /binaries/datadog-php-tracer*.tar.gz --appsec-file /binaries/dd-appsec-php-*.tar.gz)
+
+PHP_INI_SCAN_DIR="/etc/php" php /tmp/dd-library-php-setup.php \
+    "${INSTALLER_ARGS[@]}"\
+    --php-bin all
+
+export DD_APPSEC_ENABLED=1
+
+php -d error_reporting='' -d extension=ddtrace.so -d extension=ddappsec.so -r 'echo phpversion("ddtrace");' > \
+  ./LIBRARY_VERSION
+
+php -d error_reporting='' -d extension=ddtrace.so -d extension=ddappsec.so -r 'echo phpversion("ddappsec");' > \
+  ./PHP_APPSEC_VERSION
```

Comment:
Is there a specific reason why there is `-d error_reporting='' -d extension=ddtrace.so -d extension=ddappsec.so` at all? The error presumably is about the extension already being loaded, which it is, thanks to the installer.